### PR TITLE
improve perfomance for big data block transport

### PR DESCRIPTION
**Pull request by mond77**

### CODE REVIEW

**Comment by mond77** [comment]
**without** data bench:

Screen Shot 2023-02-22 at 14 15 43

**with** data_size bench: 

Screen Shot 2023-02-22 at 14 19 30

File: src/rpc/protocol.rs [hunk 1]
```diff
@@ -7,6 +7,7 @@ use std::time;
 pub const MAX_FILENAME_LENGTH: usize = 4096;
 pub const MAX_DATA_LENGTH: usize = 65536;
 pub const MAX_METADATA_LENGTH: usize = 4096;
+pub const MAX_COPY_LENGTH: usize = 1024 * 8;
```
Comment (luanshaotong) [inline src/rpc/protocol.rs:10]: I think this value should be larger to ensure a steady performance increase.

Comment (mond77) [reply src/rpc/protocol.rs:10]: why

Comment (luanshaotong) [reply src/rpc/protocol.rs:10]: Just multiply with 4? Alright, the current version is OK.

**Review by luanshaotong — APPROVED** [review]
LGTM

**Pull request merged**